src/fixate/examples/tiny.py

```diff
@@ -41,3 +41,9 @@ def test(self):
 
 
 TEST_SEQUENCE = [SimpleTest(), MyTestList([ParameterisedTest(1), ParameterisedTest(2)])]
+
+if __name__ == "__main__":
+    import fixate.__main__
+
+    argv = ["-p", __file__, "--serial_number", "1234567890"]
+    fixate.__main__.run_main_program(__file__, argv)
```

Comment:
How about having a nicer alias for that function?

e.g. adding the following to the top level `__init__.py`
`from fixate.__main__ import run_main_program as run`

So then the snippit above would become
```python
if __name__ == "__main__":
    import fixate

    argv = ["-p", __file__, "--serial_number", "1234567890"]
    fixate.run(__file__, argv)
```

Reply:
Done. Now the only thing bugging me on that is that with

```
argv = ["-p", __file__, "--serial_number", "1234567890"]
fixate.run(__file__, argv)
```

we are actually supplying the test file twice: as both `test_script_path` arg and as the script itself to run with `-p xyz.py`. It still works if we skip `test_script_path` but _not_ if we don't provide the `-p path`

`__main__.py:283-284,296-298` has:

```python
            if self.test_script_path is None:
                self.test_script_path = self.args.path
            # [… snip …]
            # Load test suite
            test_suite = load_test_suite(
                self.args.path, self.args.zip, self.args.zip_selector
            )
```

where `self.test_script_path` is not used further.  Am I on the right path (pun intended) with thinking that the `load_test_suite()` call should have `self.test_script_path` not `self.args.path` passed?

Reply:
> Am I on the right path (pun intended) with thinking that the load_test_suite() call should have self.test_script_path not self.args.path passed?

Looks like it. But much of that code is a mess that could be simplified. I've made a few passes over it in the past, but never just got stuck in a properly refactored. I'm not super keen to try fix all this right now. At least not without some kind of big picture goal. Which at the moment, is the `atdev`